File: src/main/java/com/javaToKotlin/lec01/Lec01Main.java
package com.javaToKotlin.lec01;

import java.util.ArrayList;
import java.util.List;

public class Lec01Main {

    public static void main(String[] args) {
        long number1 = 10L;         // (1)
        final long number2 = 10L;   // (2)

        Long number3 = 1_000L;      // (3)
        Person person = new Person("박현민"); // (4)

        final List<Integer> numbers = new ArrayList<>();  // collection 자체는 바꾸지 못하지만 값 추가 가능
        numbers.add(3);
    }

}

File: src/main/java/com/javaToKotlin/lec01/Lec01Main.kt
package com.javaToKotlin.lec01

fun main() {
    var number1: Long = 10L      // 바꿀 수 있는 변수
    val number2: Long = 10L      // 바꿀 수 없는 변수

    var number3: Long
//    println(number3)  // complile error

    val number4: Long
    number4 = 10L


    var number5: Long? = 1_000L
    number5 = null

    var person = Person("박현민")
}
